website/content/docs/secrets/kmip.mdx

```diff
@@ -114,6 +125,7 @@ operation_modify_attribute
 operation_query
 operation_register
 operation_rekey
+operation_rekey_keypair
 operation_revoke
```

Comment:
Please also add the entries for Sign, Signature Verify, MAC, MAC Verify, RNG Seed and RNG Retrieve.

Reply:
Done